FastBank.Domain/MessageType.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FastBank.Domain
+{
+    public enum MessageType
+    {
+        Inquery,
+    }
```

Comment:
Put this in Message.cs. It is related to the message class and it is good to be there.